srcpkgs/BAND/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'BAND'
+pkgname=BAND
+version=1.10.9
+revision=1
+archs="x86_64"
+hostmakedepends="binutils tar xz gzip"
+depends="gtk+3 libnotify nss libXScrnSaver libXtst xdg-utils at-spi2-core libuuid libappindicator libsecret"
+short_desc="Group Chat Client for Sports, School, Campus Groups, and more"
+maintainer="Adam Pschorr <adampschorr13@aol.com>"
+license="custom:Proprietary"
+homepage="https://band.us/home"
+distfiles="https://ssl.pstatic.net/cmstatic/desktop/v${version}/BAND-${version}-amd64.deb"
```

Comment:
I wasn't able to find *any* download link on the homepage. Do versions for other architectures exist? If so, it would be nice to include them too.